kompendium-oas/build.gradle.kts

```diff
@@ -17,7 +17,15 @@ sourdough {
 }
 
 dependencies {
+  val kotestVersion: String by project
+
   implementation(group = "org.jetbrains.kotlinx", "kotlinx-serialization-json", version = "1.3.2")
+
+  testImplementation(group = "io.kotest", name = "kotest-runner-junit5-jvm", version = kotestVersion)
+  testImplementation(group = "io.kotest", name = "kotest-assertions-core-jvm", version = kotestVersion)
+  testImplementation(group = "io.kotest", name = "kotest-property-jvm", version = kotestVersion)
+  testImplementation(group = "io.kotest", name = "kotest-assertions-json-jvm", version = kotestVersion)
+  testImplementation(group = "io.kotest", name = "kotest-runner-junit5-jvm", version = kotestVersion)
```

Comment:
I don't really know if there is much value having tests in the oas module, since there is not really any business logic that is performed in this module.  

Instead, can you add a test inside the `kompendium-core` showcasing this feature being used in a real example? The `KompendiumTest` class contains a large number of examples of how Kompendium tests features.

Reply:
I can work on that